biohackrxiv-paper-2022/paper-20/paper.md

```diff
@@ -0,0 +1,196 @@
+	  index: 5
+date: 11 Nov 2022
+cito-bibliography: paper.bib
+event: BioHackEU22
+biohackathon_name: "BioHackathon-Europe"
+biohackathon_url:   "https://biohackathon-europe.org/"
+biohackathon_location: "Paris, France, 2022"
+group: Project 20
+# URL to project git repo --- should contain the actual paper.md:
+git_url: https://github.com/neicnordic/sda-dashboard/tree/feature/biohackrxiv-papers/biohackrxiv-paper-2022/paper-20
+# This is the short authors description that is used at the
+# bottom of the generated paper (typically the first two authors):
+authors_short: People \emph{et al.}
+---
+
+# Introduction
+
+The European Genome-phenome Archive (EGA)[@EGA] (also known as Central EGA - cEGA) is a service for permanent archiving and sharing personally identifiable genetic and phenotypic data resulting from biomedical research projects. The Federated EGA[@FEGA], consisting of the Central and Federated EGA nodes, will be a distributed network of repositories for sharing human -omics data and phenotypes. Each node of the federation is responsible for its own infrastructure and the connection to the Central EGA. Currently, the adoption and deployment of a new federated node is challenging due to the complexity of the project and the diversity of technological solutions used, in order to ensure the secure archiving of the data and the transfer of the information between the nodes.
+
+The goal of this project was to develop an onboarding suite consisting of simple scripts, supplemented by documentation, that would help newcomers to the EGA federation in order understand in depth the main concepts, while enabling them to get involved in the development of the technology as quickly as possible.
+
+At the same time we aimed to identify existing technologies and standards
+across FEGA nodes that can be used as a reference to upcoming nodes.
+
+This biohackathon project is a result of a nordic collaboration under the umbrella of the Nordic e-Infrastructure Collaboration (NeIC)[@NEIC] where we've had three projects over the last 7 years, [Tryggve1](https://neic.no/tryggve1/)[@Tryggve1], [Tryggve2](https://neic.no/tryggve2/)[@Tryggve2] and Heilsa[@Heilsa]. For this reason the source code and scripts are shared in the same repository as the `Operator dashboard for controlling the NeIC Sensitive Data Archive` (Project 21) from BioHackathon-Europe 2022.
+
+# Results
+
+In order to achieve the project goals, our plan to focus on the main pipeline, handling the archiving of the data submitted by users to Federated EGA. Specifically, the goal is to create a number of scripts that would lead the user/developer through the process followed from submitting a file to archiving it and making it available for downloading. The scripts will shed light on the processes under the hood, including the messaging between the services, the records stored in the database as well as the tools used for encrypting and decrypting the data.
```

Comment:
```suggestion
In order to achieve the project goals, our plan is to focus on the main pipeline, handling the archiving of the data submitted by users to Federated EGA. Specifically, the goal is to create a number of scripts that would lead the user/developer through the process followed from submitting a file to archiving it and making it available for downloading. The scripts will shed light on the processes under the hood, including the messaging between the services, the records stored in the database as well as the tools used for encrypting and decrypting the data.
```